Update PR Template (#7870)
SkriptLang/Skript: .github/pull_request_template.md

### Problem
<!--- Why is this PR necessary? What problems exist that needed solving?  --->

### Solution

> <!--- How your solution fixes the problem and a summary of the major code changes.  --->

Reviewer: ```suggestion
<!--- Explain how your solution fixes the problem and summarize the major code changes.  --->
```
My thought is that we might want to use more direct verbs

### Testing Completed

> <!--- List test scripts/unit tests and any manual testing that was performed. If no test scripts or unit test are present, explain why.  --->

Reviewer: ```suggestion
<!--- List test scripts/unit tests and any manual testing that was performed. If no test scripts or unit tests are present, explain why.  --->
```
